libgrust/libformat_parser/generic_format_parser/src/lib.rs

```diff
@@ -25,6 +25,7 @@ fn is_id_continue(c: char) -> bool {
 // Workaround for Ubuntu 18.04. The default Rust package is 1.65 (and unlikely to change I assume?), but the
 // generic format parser library uses `is_some_and` which was introduced in 1.70. So this is a reimplementation,
 // directly taken from the standard library sources
+#[allow(dead_code)]
```

Comment:
I would remove the entire trait and comment instead, this was a workaround for staying in an earlier version of Rust but you've now switched to nightly :)